UnitTest1.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Medicine;
+using NUnit.Framework;
+
+
+namespace TestProject11
+{
+    [TestFixture]
+    public class CommodityUnitTests
+    {
+        [Test]
+        public void ConstructorTest()
+        {
+            var valerian = CreateTestItems();
+
+            Assert.AreEqual("Valerian", valerian.ProductName);
+            Assert.AreEqual("123456789", valerian.ItemNumber);
+            Assert.AreEqual("10$", valerian.Prise);
+            Assert.AreEqual("None", valerian.NumberOfGoods);
+            Assert.AreEqual(Medicine.withoutRecipe, valerian.Type);

```

Comment:
Несуществующий тип данных
```suggestion
            Assert.AreEqual(Prescription.withoutRecipe, valerian.Type);

```